### DISCUSSION TITLE
Feature - dynamic resizing of thumbnails

### DISCUSSION THREAD

**Discussion by ettom** [jcorporation/myMPD#812] [Ideas]
First, thanks for this great piece of software!

It would be great if there was an option to dynamically resize the thumbnails in the Database view. Currently, 175 pixels is just wide enough that on e.g. a Pixel 5, only one column of albums is displayed (but there's a lot of unused room left). If I change the thumbnail size to 170, 2 columns are displayed. Perhaps there could be an option for how many columns to display - e.g. always display at least 2 columns and shrink the thumbnails accordingly.

**Comment by jcorporation** [comment]
A simple media query could be an option. Any other thoughts?
